tools/uart_hw_test/README.md

```diff
@@ -0,0 +1,54 @@
+# UART HW test tool
+
+This tool can be used to perform the uart HW test.
+The tool performs the following tests.
+
+UART Tx byte gap time test
+1. The target prints log on the console continuously
+2. The target sends data on the selected uart
+3. The ubuntu app checks tx packet gap using time api
+
+UART Rx loss check test
+1. The target prints log on the console continuously
+2. The ubuntu app sends data on the selected uart
+3. The target verify rx data for data loss
+
+The above tests are performed for the following baud rates.
+115200, 38400, 19200, 9600, 4800
+
+### How to USE
+1. Find the UART port to be tested. For example, on the RTK8721 board, we will perform uart test for the uart1.
+2. Connect the uart (Rx, Tx, Gnd) lines from the test device to the Linux PC using Serial-USB converter.
+3. Find the port number on the device and on the PC. In our example, the uart1 appears as /dev/ttyS2 on the device. On the PC, the Serial-USB converter appears as /dev/ttyUSB2. Hence the port number is 2 in both the cases.
```

Comment:
I think 3rd step should be next of 6th. Because uart port should be input after executing the tool.

Reply:
Done.